packages/eas-cli/src/analytics/commands/common.ts

```diff
@@ -0,0 +1,24 @@
+import { Analytics, Event } from './events';
```

Comment:
nit pick: I don't think commands directory is necessary here, you could just put common.ts and event.ts directly in analytics.